dotcom-rendering/tsconfig.json

```diff
@@ -68,5 +68,14 @@
         // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
         // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     },
-    "exclude": ["cypress"]
+    "exclude": ["cypress"],
+    "ts-node": {
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
+        "compilerOptions": {
+            "module": "commonjs",
+            "esModuleInterop": true,
+            "allowSyntheticDefaultImports": true,
+        }
+    },
```

Comment:
Did you have much success getting ts-node to work on DCR? I think when @mxdvl tried we couldn't get it to work!